.gitignore

```diff
@@ -1,3 +1,5 @@
+config.py
```

Comment:
Thanks for the PR! The change to generals.py is good, but please could you undo the change to .gitignore?